package.json

```diff
@@ -6,24 +6,23 @@
   "changelogUrl": "https://github.com/DeNA/Anjin/releases",
   "dependencies": {
     "com.cysharp.unitask": "2.3.3",
-    "com.nowsprinting.test-helper.monkey": "0.11.0",
-    "com.nowsprinting.test-helper.random": "0.3.0",
```

Comment:
@nowsprinting 
Sorry for my late reply. Is the deletion of this line intentional?

Reply:
Yes. test-helper.random is already depends via test-helper.monkey.

see: https://github.com/nowsprinting/test-helper.monkey/blob/master/package.json#L11